LondonBritishLibrary/orient/BLorient545.xml

```diff
@@ -31,21 +32,225 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                         </altIdentifier>
                         <altIdentifier>
                             <idno>Wright 132</idno>
-                            
+
                         </altIdentifier>
-                        
+
                     </msIdentifier>
                     <msContents>
-                        <summary/>
-                        
-                        
+<msItem xml:id="ms_i1">
+<locus from="3r"/>
+<title type="complete" xml:lang="en" ref="LIT1960Mashaf">Missal of the Ethiopian Church</title>
+<textLang xml:lang="gez"/>
+</msItem>
+
```

Comment:
All textual units which follow fall within the Missal, right? So, they should be nested within the first msItem and numbered as 1.1 (1.1.1), 1.2, and so on. So, move the closing tag at the place where the Missal really ends, and renumber other msItems